include/mxnet/lib_api.h

```diff
@@ -997,7 +1015,8 @@ typedef int (*opCallFComp_t)(fcomp_t fcomp, const char* const* keys,
                              void** in_indices, void** out_indices,
                              void** in_indptr, void** out_indptr,
                              int64_t* in_indices_shapes, int64_t* out_indices_shapes,
-                             int64_t* in_indptr_shapes, int64_t* out_indptr_shapes);
+                             int64_t* in_indptr_shapes, int64_t* out_indptr_shapes,
+                             void* cpu_states, void* gpu_states);
```

Comment:
can we call these rand_ cpu_states and rand_gpu_states so its more clear that these are for RNG?

Reply:
its still called cpu_states here, can we rename it here too?